docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md

```diff
@@ -0,0 +1,78 @@
+---
+type: docs
+title: "How-To: Configure the Azure cloud provider with Workload identity"
+linkTitle: "Azure provider with Workload identity"
+description: "Learn how to configure the Azure provider with Workload identity for your Radius Environment"
+weight: 200
+categories: "How-To"
+tags: ["Azure"]
+---
+
+The Azure provider allows you to deploy and connect to Azure resources from a self-hosted Radius Environment. It can be configured:
+
+- [Interactively via `rad init`](#interactive-configuration)
+- [Manually via `rad env update` and `rad credential register`](#manual-configuration)
+
+## Prerequisites
+
+- [Azure subscription](https://azure.com)
+- [az CLI](https://aka.ms/azcli)
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+- [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/installation.html) installed in your cluster, including the [Mutating Admission Webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
+-	Create an app registration at Microsoft Entra ID 
+-	Configure the federated credential for Radius components to deploy Azure resources following the script [here](insertscript) or manually configure the federated credential following the steps [here](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azp#kubernetes) 
```

Comment:
@willtsai - can you help add the details of the service account and namespace here